tests/retriveAlreadyBookedActions.test.js

```diff
@@ -0,0 +1,60 @@
+const chai = require("chai")
+
+describe("retriveAlreadyBookedDays module actions", (done) => {
+    const daysOff = [
+        {"id": 0,
+         "userId": 0,
+         "daysOff": [2018-06-04, 2018-06-05]
+        },
+
+        {"id": 1,
+         "userId": 1,
+         "daysOff": [2018-06-20, 2018-06-21]
+        },
+
+        {"id": 2,
+         "userId": 0,
+         "daysOff": [2018-08-23, 2018-08-24, 2018-08-27, 2018-08-28]
+        }
+    ]
+    describe("findBookedDays function tests", done => {
+
+        it("Should return an array with the days already booked by user with id 0", done => {
+            const { findBookedDays } = require("../src/actions/retriveAlreadyBookedDaysActions");
+            const input = 0
```

Comment:
It's usually preferred to call variables in a way that explains the value it holds

Reply:
i renamed them